src/utils/EvmUtils.ts

```diff
@@ -87,4 +89,22 @@ export class EvmUtils {
     public static ethersResultToObjectRecursive(result: ethers.Result) {
         return Codec.ethersResultToObjectRecursive(result);
     }
+
+    public static encodeExitChannelBlock(ecb: ExitChannelBlockStruct): string {
+        return Codec.encode(ecb, Type.ExitChannelBlock);
```

Comment:
@daiagi can we remove all of these explicit functions for specific types which call encode/decode and just have encode/decode

Reply:
Yes. Agree

Reply:
> Yes. Agree

i will remove my changes, do you want to open a seperate PR for this ?